src/keypair.js

```diff
@@ -90,15 +90,21 @@ export class Keypair {
 
   /**
    * Returns `Keypair` object representing network master key.
+   * @param {string} [networkPassphrase] passphrase of the target stellar network (e.g. "Public Global Stellar Network ; September 2015").
    * @returns {Keypair}
    */
-  static master() {
-    if (Network.current() === null) {
+  static master(networkPassphrase) {
+    // Deprecation warning. TODO: remove optionality with next major release.
+    if (!networkPassphrase) {
+      console.warn('Global `Network.current()` is deprecated. Please pass explicit argument instead, e.g. `Keypair.master(Networks.PUBLIC)`.')
+      networkPassphrase = Network.current()
```

Comment:
@Akuukis shouldn't this be `Network.current()..networkId()` ?

Reply:
Yes. I think this PR is not relevant anymore as opinions converge to 2nd solution of the problem. Over at other PR (#207) this problem is already fixed.